core/templates/dev/head/pages/exploration_editor/statistics_tab/StatisticsTab.js

```diff
@@ -223,6 +210,17 @@ oppia.controller('StatisticsTab', [
                 legendPosition: 'right',
                 width: 240
               };
+
+              var title1 = 'Answer feedback statistics';
+              $scope.COMPLETION_RATE_PIE_CHART_OPTIONS1 = (
```

Comment:
Don't you need an angular.copy() to break references to the object? Otherwise if I'm reading correctly the stuff below is modifying the same object as this line is.

Reply:
Oh yes, that used to happen in the backend - missed it when doing it here. Done.